### PR TITLE
[WIP] Supports do not balance region to the specified store

### PR DESCRIPTION
**Pull request by loxp**
UCP #2176

### What problem does this PR solve? 

Supports do not balance region to the specified store

### Check List 

Tests 

 - No code

Code changes

- Has persistent data change

Side effects

Related changes

 - Need to update the documentation
 - Need to be included in the release notes

### CODE REVIEW

**Comment by disksing** [comment]
/cc @nolouch Please take a look.
Is it we want? Do we want to stop all schedulers or only balance-region?

File: server/schedulers/balance_region.go [hunk 1]
```diff
@@ -94,6 +96,9 @@ func newBalanceRegionScheduler(opController *schedule.OperatorController, conf *
 		filter.StoreStateFilter{ActionScope: scheduler.GetName(), MoveRegion: true},
 		filter.NewSpecialUseFilter(scheduler.GetName()),
 	}
+	scheduler.targetFilters = []filter.Filter{
+		filter.NewLabelConstaintFilter("", []placement.LabelConstraint{{Key: "store-do-not-balance-to", Op: placement.NotExists}}),
```
Comment (nolouch) [inline server/schedulers/balance_region.go]: This method is not recommended, you can add parameters to this schedule, refer to https://github.com/pingcap/pd/pull/1937.

Comment (nolouch) [inline server/schedulers/balance_region.go]: Hi, @loxp , you can use this way: https://github.com/pingcap/pd/pull/2150/

Comment (loxp) [reply server/schedulers/balance_region.go]: OK

**Comment by nolouch** [comment]
closed due to finished.

**Pull request closed without merging**